tests/utils/clients/graphql.py

```diff
@@ -71,6 +74,7 @@ def graphql_server():
             debug=app.debug,
         )
 
+        dispose_context()
```

Comment:
Yes, it is. Even though it should be overwritten with a new request, just to avoid any side-effects (like try to write some code out of request scope ) I think it's nice to clean it up